QQE-378.md

```diff
@@ -0,0 +1,52 @@
+# QQE-378 - HTTP module / REST use-cases - Development topic compression Brotli4j
+## Links
+Jira : [QQE-378](https://issues.redhat.com/browse/QQE-378)
+
+PR:  TODO
```

Comment:
Can TODO already be replaced with the https://github.com/quarkus-qe/quarkus-test-suite/pull/1717 or are there are PRs connected to this TP? If not, we can probably make this TP ready for review?